### ISSUE TITLE
Improve text contrast ratios

### ISSUE THREAD

**Issue by chrismilway** [plone/plonetheme.barceloneta#24]
There is a problem with contrast ratios for text in this theme.

In some cases the text flat out fails to reach the required 4.5:1 ratio required by WCAG AA (paragraph element with the 'discreet' class)

In other cases the use of lighter font weights (100 and 300) are causing, on mac at least, the stated text colour value of the text to render much lighter (for example portlet text and the breadcrumbs, again pushing the contrast ratio below required levels.

Screenshot:
![plone5](https://cloud.githubusercontent.com/assets/4137130/7700835/7ead310e-fe19-11e4-8d45-16984583dd52.png)

I suggest you don't use the lighter weights for text sizes and/or use darker colour values.

p.s. I have made the assumption that meeting accessibility requirements is still important to Plone. I have used criteria from WCAG AA because that's what my clients usually ask for, and it seems to me that a default theme should set a good example.

**Comment by bloodbare** [comment]
+1 on this idea , @albertcasado  can you check a bit this accessibility ?
